pennylane/operation.py

```diff
@@ -879,12 +887,12 @@ def kraus_matrices(self):
         Returns:
             list(array): list of Kraus matrices
         """
-        return self._kraus_matrices(*self.parameters)
+        return self._kraus_matrices(*self.data)
```

Comment:
Will have to stay such that trainable parameters can also be passed
```suggestion
        return self._kraus_matrices(*self.parameters)
```

Reply:
Ah yes! forgot to change this! Thanks!